pkgs/top-level/all-packages.nix

```diff
@@ -17793,7 +17793,9 @@ with pkgs;
   coq_8_4 = coqPackages_8_4.coq;
   coq_8_5 = coqPackages_8_5.coq;
   coq_8_6 = coqPackages_8_6.coq;
-  coq_HEAD = callPackage ../applications/science/logic/coq/HEAD.nix {};
+  coq_HEAD = callPackage ../applications/science/logic/coq {
+    version = "8.7pre-2017-05-28";
```

Comment:
@rht If there was a tag already I would have used it instead but there is not and this is precisely the point of pre-releases.
Though, if you have a look at the mailing lists such as Coq-Club or Coq-Dev, you'll see that developers are already calling the development version 8.7. More importantly, calling it 8.6-unstable would be plain wrong because the 8.6 branch is still being developed but only regarding bugfixes, while this package really corresponds to what Coq 8.7 will be (modulo a few features still missing - that's why we're not in beta yet - and a lot of bugs to fix).